src/main/java/study/racing/domain/Car.java

```diff
@@ -3,38 +3,55 @@
 import java.util.Objects;
 
 public class Car {
-    private static final int DEFAULT_POSITION = 0;
-    private static final int MOVE_CONDITION = 4;
-    private int position;
+    private final CarName name;
+    private CarPosition position;
 
-    public Car() {
-        this.position = DEFAULT_POSITION;
+    public Car(String name) {
+        this.name = new CarName(name);
+        this.position = new CarPosition();
```

Comment:
```suggestion
        this(name, 0);
```
부 생성자에서는 필드 값을 초기화하지 않는 것을 추천
필드 값 초기화는 주 생성자에서만

Reply:
처음에 이렇게 작성하였다가 0이라는 상수가 생기는것 같아서 필드값을 초기화하도록 수정했네요.
 부 생성자에서 필드값을 초기화하지 않는다는 내용은 어제도 나왔던 내용이였는데 놓쳤군요